stdlib/unittest/case.pyi

```diff
@@ -289,6 +290,16 @@ class TestCase:
         # Runtime has *args, **kwargs, but will error if any are supplied
         def __init_subclass__(cls, *args: Never, **kwargs: Never) -> None: ...
 
+    if sys.version_info >= (3, 14):
+        def assertIsSubclass(self, cls: type[Any], superclass: type[Any] | tuple[type[Any], ...], msg: Any = None) -> None: ...
+        def assertNotIsSubclass(self, cls: type[Any], superclass: type[Any] | tuple[type[Any], ...], msg: Any = None) -> None: ...
```

Comment:
```suggestion
        def assertIsSubclass(self, cls: type, superclass: type | tuple[type, ...], msg: Any = None) -> None: ...
        def assertNotIsSubclass(self, cls: type, superclass: type | tuple[type, ...], msg: Any = None) -> None: ...
```

(We would need an explanation for `msg: Any`, but considering that other assert methods also use this annotation, we could probably at some point introduce a type alias for these arguments.)